File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1116.py
n = int(input())

for i in range(0, n):
    ns = input().split(' ')
    x = int(ns[0])
    y = int(ns[1])
    if y==0:
        print("divisao impossivel")
    else:
        print("{:.1f}".format(x/y))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1116.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = parseInt(lines.shift());
        
for(i=0;i<n;i++){
    ns = lines.shift().split(' ')
    x = parseInt(ns[0])
    y = parseInt(ns[1])
    if(y==0){
        console.log("divisao impossivel");
    } else{
        console.log((x/y).toFixed(1));
    }
}